src/overlay.scss

```diff
@@ -11,6 +11,7 @@ $block: #{$fd-namespace}-overlay;
   $fd-overlay-background-color: rgba(#4a505c, 0.3) !default;
 
   --fd-overlay-background-color: #{$fd-overlay-background-color};
+  $fd-overlay-message-strip-top-margin: fd-space(7);
```

Comment:
Can you revert any changes in this file?

Reply:
These are the changes needed for messages strip to work.